cumulusci.yml

```diff
@@ -48,69 +48,76 @@ tasks:
                 TDTM_Global_API.setTdtmConfig( TDTM_Global_API.getTdtmConfig() );
 
     execute_qa_apex:
-        description: Runs Apex setup for QA orgs.
+        # description: Runs anonymous apex for qa to enable course connections.
```

Comment:
```suggestion
        description: Runs anonymous apex for qa to enable course connections.
```

Reply:
I think the proposed new description here is invalid. It does other things now besides enabling course connections, right?